README.md

```diff
@@ -1,8 +1,23 @@
 # Galasa Integration Tests
-## Project Status
-| Project | Gate | Code Smells | 
-| ------- | ---- | ----------- | 
-| Parent | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-inttests-parent&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-inttests-parent) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-inttests-parent&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-inttests-parent) |
-| Tests | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.inttests&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.inttests) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.inttests&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.inttests) |
-| OBR | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.inttests.obr&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.inttests.obr) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.inttests.obr&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.inttests.obr) |
+This repository contains end-to-end tests for Galasa rather than the Manager-specific IVT (Installation Verification Test) tests. Includes code that enables you to run a Galasa test from a command line or from Docker.
```

Comment:
Um, no it doesn't contain "Includes code that enables you to run a Galasa test from a command line or from Docker."

This repo contains all the Integration tests necessary to test Galasa itself.